### PR TITLE
new (mp) : New Cisco Callmanager SXML Monitoring Procedure [FR]

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Prérequis
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Cisco Callmanager :
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:40]: Uppercase C

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+```bash
+yum install centreon-plugin-Network-Cisco-Callmanager-Sxml.noarch
+```
+
+2. Sur l'interface Web de Centreon, installer le Plugin-Pack *Cisco Callmanager SXML* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--Offline IMP License-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Cisco Callmanager :
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:50]: Uppercase C

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+| :---------- | :----------------------- | :-------------------------------------------------- |
+| X           | CUCMSXMLAPIPORT          | Port used. Default is 8443                          |          
+| X           | CUCMSXMLAPIPROTO         | Protocol used. Default is https                     |
+|             | CUCMSXMLAPIEXTRAOPTIONS  | Any extra option you may want to add to the command |
+
+## FAQ
+
+### Comment puis-je tester le Plugin et que signifient les options des commandes ?
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:82]: Uppercase C

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+La commande ci-dessus contrôle les alertes de Cisco Callmanager via la SXML (```--mode=alerts```).
+Le Plugin utilise l'api-username (```--api-username='centreoncucm'```), l'api-password (```--api-password='myapipassword'```)
+et il se connecte à l'hôte _mycucm.com_ (```--hostname='mycucm.com'```) 
+sur le port _443_ (```--port='443'```) utilisant le protocol _https_ (```--proto='https'```).
+Par defaut le méthode du backend est _curl_ (```--http-backend=curl```).
+
+Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+
+```bash
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:114]: 1 option / line

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+#### ```UNKNOWN: 501 Protocol scheme 'connect' is not supported |``` 
+
+Suite à la mise en place du proxy, j'obtiens le message suivant ```UNKNOWN: 501 Protocol scheme 'connect' is not supported |```
+Cela signifie que le protocole de connexion au proxy n'est pas supporté par la libraire *LWP* utlisée par défaut par le Plugin Centreon.
+Cette erreur peut être résolue en utilisant le backend HTTP *lwp*. Pour ce faire, ajoutez l'option ```--http-backend='lwp'``` à la commande.
+
+### Comment puis-je supprimer les perfdatas *count* dans le cas où je ne souhaite vérifier qu'une seule application ?
+
+Le Plugin permet de filtrer sur un ou plusieurs éléments mais permet également de récupérer l'ensemble des éléments si aucun filtre n'est spécifié.
+De ce fait, des perfdatas "globales" sur les statistiques des objets sont ajoutées par défaut. Il est possible de supprimer ces données de performance en appliquant le filtre suivant: ```--filter-perfdata='^$'```.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:136]: Copy / Paste ?

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 1 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md]: Uppercase C => OK ✔️

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 2 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md]: Uppercase C => OK ✔️

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 3 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md]: Uppercase C => OK ✔️

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 4 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md:114]: Added 1 option / line

File: fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md (hunk 5 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-cisco-callmanager-sxml.md]: Senteces removed

**Review by Sims24 — APPROVED** [review]

**Pull request merged**